### PR TITLE
Alias the vip User ID with the anonymized id in Tracks

### LINKED ISSUE #316: Tracks: update anonymized ID
When a user first runs `vip`, we assign an anonymized ID. When they log in, we should send the de-anonymize event to Tracks with their VIP user ID (for more useful tracking across CLI/API/Dash). Will need to check with the Tracks team the specs of the event.

### PR DESCRIPTION
**Pull request by dchymko**
On user login, associate the vip user id with the tracks anonymous id. This will allow better tracking of user events across various services.

Fixes #316

### CODE REVIEW

File: src/bin/vip.js [hunk 1]
```diff
@@ -115,6 +115,9 @@ const rootCmd = async function() {
 			throw e;
 		}
 
+		//de-anonymize user for tracking
+		await aliasUser( token.id );
```
Comment (mjangda) [inline src/bin/vip.js:119]: We'll need to update the `uuid` in `lib/token` as well, otherwise we'll keep sending the anon ID with future events.

Comment (dchymko) [reply src/bin/vip.js:119]: Okay, I think the following commit addresses that.

**Comment by dchymko** [comment]
This appears to be working in tracks correctly. If I login, I see the aliasUser event, and then subsequent events have my userID instead of the anonymized one.

**Review by mjangda — APPROVED** [review]
🚢

**Pull request merged**